examples/quickstart/jupyter-notebooks/notebooks/01-introduction/02-datagen-intro.ipynb

```diff
@@ -0,0 +1,540 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9e07b3f5-d919-4179-91a1-0f6b66c42757",
+   "metadata": {},
+   "source": [
+    "# Data Generator Server\n",
+    "The default docker compose deployment includes a data generation service created from the published docker image `imply/datagen:latest`. \n",
```

Comment:
```suggestion
 "The default Docker Compose deployment includes a data generation service created from the published Docker image at `imply/datagen:latest`. \n",
```

We should offer an option for them to build the data generator locally if they don't want to pull an image from a third-party repo.